pxtblocks/fields/field_note.ts

```diff
@@ -330,6 +330,31 @@ export class FieldNote extends Blockly.FieldNumber implements FieldCustom {
 
         Blockly.DropDownDiv.setColour(this.primaryColour, this.borderColour);
         Blockly.DropDownDiv.showPositionedByBlock(this, this.sourceBlock_ as Blockly.BlockSvg, () => this.onHide());
+
+
+        this.htmlInput_.addEventListener("keydown", this.keyHandler);
```

Comment:
We should remove this. You can add the widgetDispose_ method and call removeEventListener there before calling super. I.e., this:

```
    protected widgetDispose_(): void {
        this.htmlInput_.addEventListener("keydown", this.keyHandler);
        super.widgetDispose_();
    }
```